frontend/src/features/projects/common/components/ProjectFinancialTable.tsx

```diff
@@ -33,15 +48,29 @@ const ProjectFinancialTable = ({ disabled }: { disabled: boolean }) => {
               formikProps={context}
               disabled={disabled}
             />
-          </Row>
-          <Row>
+          </Form.Row>
+          <Form.Row>
             <Form.Label column md={4}>
               Appraised Value
             </Form.Label>
             <FastCurrencyInput field="appraised" formikProps={context} disabled={disabled} />
-          </Row>
+          </Form.Row>
+        </Col>
+        <Col md={4}>
+          <Form.Row>
+            <Form.Label column md={4}>
```

Comment:
This would have been my error from when I added the original 4 fields, I can include it in a later PR - but wondering if we should add `margin-bottom: 5px` to the labels so they align with the inputs a bit better

Reply:
What's the best way to do this?  I don't like adding a style manually to each.

Reply:
probably create a styled component wrapping Form.Label with the margin-bottom.

Reply:
@Fosol not seeing the requested change here, or a TODO. My concern is that this'll get forgotten if we don't at least have a tech-debt item for it.

Reply:
Spoke with jonny and he mentioned we can just correct the shared css instead.  That was part of the PR as well.

Reply:
thanks,sounds good.